packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/test/handler.test.ts

```diff
@@ -458,63 +247,66 @@ describe('SEPConfiguratorResource', () => {
       mockSpotEventPluginClient.addGroups = mockAddGroups;
 
       // WHEN
-      await handler.doCreate('physicalId', allConfigs);
+      await handler.doCreate('physicalId', validSepConfiguration);
 
       // THEN
       expect(mockAddGroups.mock.calls.length).toBe(1);
-      expect(mockAddGroups).toHaveBeenCalledWith(deadlineGroups);
+      expect(mockAddGroups).toHaveBeenCalledWith([deadlineGroup]);
     });
 
     test('create pools', async () => {
       // GIVEN
-      const mockAddPools = jest.fn( (a) => returnTrue(a) );
+      const mockAddPools = jest.fn( (_a) => Promise.resolve(true) );
       mockSpotEventPluginClient.addPools = mockAddPools;
 
       // WHEN
-      await handler.doCreate('physicalId', allConfigs);
+      await handler.doCreate('physicalId', validSepConfiguration);
 
       // THEN
       expect(mockAddPools.mock.calls.length).toBe(1);
-      expect(mockAddPools).toHaveBeenCalledWith(deadlinePools);
+      expect(mockAddPools).toHaveBeenCalledWith([deadlinePool]);
     });
 
     test('throw when cannot add groups', async () => {
       // GIVEN
-      mockSpotEventPluginClient.addGroups = jest.fn( (a) => returnFalse(a) );
+      mockSpotEventPluginClient.addGroups = jest.fn( (_a) => Promise.resolve(false) );
 
       // WHEN
-      const promise = handler.doCreate('physicalId', allConfigs);
+      const promise = handler.doCreate('physicalId', validSepConfiguration);
 
       // THEN
       await expect(promise)
         .rejects
-        .toThrowError(`Failed to add Deadline group(s) ${allConfigs.deadlineGroups}`);
+        .toThrowError(`Failed to add Deadline group(s) ${validSepConfiguration.deadlineGroups}`);
     });
 
     test('throw when cannot add pools', async () => {
       // GIVEN
-      mockSpotEventPluginClient.addPools = jest.fn( (a) => returnFalse(a) );
+      mockSpotEventPluginClient.addPools = jest.fn( (_a) => Promise.resolve(false) );
 
       // WHEN
-      const promise = handler.doCreate('physicalId', allConfigs);
+      const promise = handler.doCreate('physicalId', validSepConfiguration);
 
       // THEN
       await expect(promise)
         .rejects
-        .toThrowError(`Failed to add Deadline pool(s) ${allConfigs.deadlinePools}`);
+        .toThrowError(`Failed to add Deadline pool(s) ${validSepConfiguration.deadlinePools}`);
     });
 
     test('throw when cannot save spot fleet request configs', async () => {
       // GIVEN
-      const mockSaveServerData = jest.fn( (a) => returnFalse(a) );
+      const mockSaveServerData = jest.fn( (_a) => Promise.resolve(false) );
       mockSpotEventPluginClient.saveServerData = mockSaveServerData;
 
       // WHEN
-      const promise = handler.doCreate('physicalId', noPluginConfigs);
+      const promise = handler.doCreate('physicalId', {
+        connection: validSepConfiguration.connection,
+        spotFleetRequestConfigurations: validSepConfiguration.spotFleetRequestConfigurations,
+      });
```

Comment:
I might be missing something, but can we simplify this to:

```suggestion
      const promise = handler.doCreate('physicalId', validSepConfiguration);
```